src/registry/coverage.ts

```diff
@@ -21,12 +21,17 @@ const getProxiedOptions = (url: string): OptionsOfTextResponseBody => ({
   url,
 });
 
-export const getCurrentApiVersion = async (): Promise<number> =>
-  (
-    await got(getProxiedOptions('https://mdcoverage.secure.force.com/services/apexrest/report')).json<{
-      versions: { selected: number };
-    }>()
-  ).versions.selected;
+type ApiVersion = {
+  label: string;
+  url: string;
+  version: string;
+};
+
+export const getCurrentApiVersion = async (): Promise<number> => {
+  const apiVersionsUrl = 'https://dx-extended-coverage.my.salesforce-sites.com/services/data';
```

Comment:
If we're taking that approach, maybe `https://org62.my.salesforce-sites.com/services/data` would be more reliable/scalable than this team's endpoint.

or use `Promise.any` of several known URLs